nabla-lib/network/network_linux.go

```diff
@@ -234,28 +271,11 @@ func CreateTapInterfaceDocker(tapName string, master string) (
 		return nil, nil, nil,
 			fmt.Errorf("no master interface: %v", err)
 	}
-
-	addrs, err := netlink.AddrList(masterLink, netlink.FAMILY_V4)
+	// add bhere
```

Comment:
typo?

Reply:
yea i put a place holder here so i knew where to put the refactored function, i'll fix that.